packages/app/src/specs/InlineRunAllSpecs.cy.tsx

```diff
@@ -23,6 +25,15 @@ describe('<InlineRunAllSpecs/>', () => {
         cy.findByTestId('tooltip-content').should('contain.text', 'Run 40 specs')
       })
     })
+
+    it('emits expected event on click and with keyboard', () => {
+      cy.get('button')
+      .click()
+      .type(' ')
+      .type('{enter}')
+
+      cy.get('@runAllStub').should('have.been.calledThrice')
```

Comment:
Can we just create a fresh stub? This test relies on other tests to pass - deleting or changing the order of those would break this test, which isn't good - I think the tests should be able to run in any order, independently of each other.